tests/test_annotated_tlr_dataset_conversion.py

```diff
@@ -39,182 +39,9 @@ def t4_dataset_path():
     shutil.rmtree(t4_output_base, ignore_errors=True)
 
 
-@pytest.fixture
-def attribute_list():
-    with open(TEST_CONFIG_ROOT_DIR / "label/attribute.yaml") as f:
-        arrtibute_dict = yaml.safe_load(f)
-        arrtibute_list = []
-        for k, v in arrtibute_dict.items():
-            for key in v.keys():
-                arrtibute_list.append(f"{k}.{key}")
+def test_annotated_tlr_dataset_diff(t4_dataset_path):
+    """Test that generated annotated T4 TLR dataset matches expected output."""
+    generated_path = Path(t4_dataset_path)
+    expected_path = Path(str(generated_path).replace("_generated", ""))
```

Comment:
[NITS] It looks better to avoid casting back
```suggestion
    expected_path = Path(t4_dataset_path.replace("_generated", ""))
```